### ISSUE TITLE
Dropout modification in nnU-Net v2

### ISSUE THREAD

**Issue by MLK97** [MIC-DKFZ/nnUNet#1342]
I have a follow-up to a question posted a few years ago (that I for some reason can't link directly so I'll post the URL instead, sorry!) [https://github.com/MIC-DKFZ/nnUNet/issues/8](Question about overfitting).

I am looking for the ```dropout_op_kwargs = {'p': 0, 'inplace': True}``` parameter or something similar in the `nnUNetTrainer.py` file but unfortunately can't find it anymore in for v2.

Has this been moved, removed or do I simply need better glasses to find it in there?

**Comment by FabianIsensee** [comment]
Hey, Dropout was never used by nnU-Net and I would not recommend using it. The implementation was also really bad with dropout being applied throughout the entire network. There are other ways we can introduce more regularization that are more promising, especially data augmentation

**Comment by FabianIsensee** [comment]
From my experience the best uncertainties are ensemble uncertainties. Train the 5 models and let them do predictions independently. Store the softmax predictions. And use the disagreements there (entropy, standard deviation) to derive uncertainties

**Comment by MLK97** [comment]
Great! Thank you very much :-)
